packages/plugins/rum-privacy/src/types.ts

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+export interface RumPrivacyOptions {
+    exclude?: RegExp[] | string[];
+    include?: RegExp[] | string[];
+    module?: 'cjs' | 'esm';
+    jsx?: boolean;
+    transformStrategy?: 'ast';
+    typescript?: boolean;
+    disabled?: boolean | undefined;
+}
```

Comment:
I'm not sure I follow, are these settings the customer can tweak?
Do they actually need this?